### PR TITLE
Core - Use json module for error_json exception type rendering instead of jinja2

### PR DESCRIPTION
**Pull request by dominikschubert**
Encountered an issue with the json serialization/parsing of exception messages because they were simply substituted in the jinja2 template instead of being properly encoded. Since there didn't seem to be any reason to keep using jinja2 for this manual json rendering, I've replaced this in favor of using the built-in json module.

### CODE REVIEW

**Review by whummer — APPROVED** [review]
Awesome, nice catch and great fixes, thanks @dominikschubert 👍 . Awaiting feedback and review from @bblommers .. 🙏 Thanks!

File: moto/core/exceptions.py [hunk 1]
```diff
@@ -53,22 +53,22 @@ class RESTError(HTTPException):
         "single_error": SINGLE_ERROR_RESPONSE,
         "wrapped_single_error": WRAPPED_SINGLE_ERROR_RESPONSE,
         "error": ERROR_RESPONSE,
-        "error_json": ERROR_JSON_RESPONSE,
```
Comment (bblommers) [inline moto/core/exceptions.py:56]: Can we remove the `ERROR_JSON_RESPONSE`-global, or is that still in use somewhere else?

**Review by bblommers — CHANGES_REQUESTED** [review]

File: moto/core/exceptions.py (hunk 1 above)
Comment (dominikschubert) [reply moto/core/exceptions.py:56]: Yeah actually thought I had deleted this already. :thinking:  Removed this now and I've also changed the exception rendering for managedblockchain in the same way.

**Review by bblommers — APPROVED** [review]
LGTM! Thanks for the fix @dominikschubert

**Pull request merged**